pytorch_lightning/plugins/io/checkpoint_plugin.py

```diff
@@ -52,3 +52,11 @@ def load_checkpoint(self, path: _PATH, storage_options: Optional[Any] = None) ->
 
         Returns: The loaded checkpoint.
         """
+
+    @abstractmethod
+    def remove_checkpoint(self, path: _PATH) -> None:
+        """Remove checkpoint filepath from the filesystem.
```

Comment:
```suggestion
        """Remove checkpoint file from the filesystem.
```
the whole file, right? :)